Surface_mesh_segmentation/doc/Surface_mesh_segmentation/Surface_Mesh_Segmentation.txt

```diff
@@ -119,16 +119,16 @@ where:
   - \f$x_f\f$ denotes the cluster assigned to facet \f$f\f$,
   - \f$P(f|x_p)\f$ denotes the probability of assigning facet \f$f\f$ to cluster \f$x_p\f$,
   - \f$\theta(f,g)\f$ denotes the dihedral angle between neighboring facets \f$f\f$ and \f$g\f$:
-         concave angles and convex angles are weighted by 1 and 0.1 respectively,
-  - \f$\epsilon\f$ denotes the minimal probability threshold,
+         concave angles and convex angles are weighted by 1 and 0.08, respectively,
+  - \f$\epsilon_1, \epsilon_2\f$ denote minimal probability and angle thresholds, respectively,
   - \f$\lambda \in [0,1]\f$ denotes a smoothness parameter.
          </td>
   </tr>
   </table>
 
 Note both terms of the energy function, \f$ e_1 \f$ and \f$ e_2 \f$, are always non-negative.
 The first term of the energy function provides the contribution of the soft clustering probabilities.
-The second term of the energy function is a geometric criterion that is larger when two adjacent facets sharing a sharp and concave edge are not in the same cluster.
+The second term of the energy function is a geometric criterion that is larger the larger the dihedral angle between two adjacent facets not in the same cluster.
```

Comment:
I've rephrased it a bit, how does this look?

Reply:
Except if I'm wrong you have a greater contribution the closer you are to 180 degrees. I guess that's what you meant but largest angle could be 360 degree.

Reply:
You're quite right. I've reworded it again and included the valid range for theta in the description of the expression symbols.